content/faq/_index.md

```diff
@@ -85,3 +85,8 @@ compile('io.springfox:springfox-swagger2:${swagger2.version}') {
 {{< /prettify >}}
 
 {{% /faq_question %}}
+
+{{% faq_question "How to map protobuf classes?" %}}
+For MapStruct to work with protobuf generated classes one needs to use a custom `AccessorNamingStrategy`.
```

Comment:
Is this specific to a version of ProtoBuf (I remember someone mentioned differences between 2 and 3)?